UMA-Bots/Wallet-Monitoring/src/agent.ts

```diff
@@ -0,0 +1,68 @@
+import {
+  Finding,
+  HandleTransaction,
+  ethers,
+  Initialize,
+  TransactionEvent,
+  getEthersProvider,
+} from "forta-agent";
+import { NetworkManager } from "forta-agent-tools";
+import { FILLED_RELAY_EVENT, getFindingInstance } from "./utils";
+import { NetworkDataInterface, NM_DATA } from "./network";
+
+const networkManagerCurr = new NetworkManager(NM_DATA);
+
+export function provideInitialize(
+  networkManager: NetworkManager<NetworkDataInterface>,
+  provider: ethers.providers.Provider
+): Initialize {
+  return async () => {
+    await networkManager.init(provider);
+  };
+}
+
+export function provideHandleTransaction(
+  filledRelayEvent: string,
+  networkManager: NetworkManager<NetworkDataInterface>
+): HandleTransaction {
+  return async (txEvent: TransactionEvent) => {
+    const findings: Finding[] = [];
+    const filledRelayEventTxns = txEvent.filterLog(
+      filledRelayEvent,
+      networkManager.get("spokePoolAddr")
+    );
+
+    filledRelayEventTxns.forEach((filledRelayEvent) => {
+      const {
+        amount,
+        originChainId,
+        destinationChainId,
+        depositor,
+        recipient,
+        isSlowRelay,
+      } = filledRelayEvent.args;
+
+      if (networkManager.get("monitoredList").indexOf(depositor) > -1) {
```

Comment:
not wrong, but this looks better for readability
```suggestion
      if (networkManager.get("monitoredList").includes(depositor)) {
```